tests/test_issues/test_validate_polymorphism.py

```diff
@@ -0,0 +1,64 @@
+import  unittest
+
+from linkml.generators.pythongen import PythonGenerator
+
+from tests.test_issues.environment import env
+from tests.utils.test_environment import TestEnvironmentTestCase
+from linkml_runtime.utils.compile_python import compile_python
+from linkml.validators import JsonSchemaDataValidator
+from linkml_runtime.utils.schemaview import SchemaDefinition, SchemaView
+from .test_pydantic_polymorphism import data_str, gen_test_schema
+import json
+
+class PydanticPolymorphismTestCase(TestEnvironmentTestCase):
```

Comment:
is this mis-named? this seems to be testing PythonGenerator (i.e dataclass generation)

Reply:
it was originally meant to test the whole linkml-validate flow  for polymorphism (which includes python generator but also json schema validation), but since the json schema validation is now a separate PR, it makes sense to rename the file. will do.